libkbfs/md_ops.go

```diff
@@ -104,6 +105,9 @@ func (md *MDOpsStandard) verifyWriterKey(ctx context.Context,
 		// extra work by downloading the same MDs twice (for those
 		// that aren't yet in the cache).  That should be so rare that
 		// it's not worth optimizing.
+		//
```

Comment:
I don't think so -- the only way it could happen is if the first update in the branch sets the copy bit, but that only happens for rekeys or finalizations, neither of which can happen while unmerged.